core/src/main/scala/cromwell/core/path/EvenBetterPathMethods.scala

```diff
@@ -96,9 +97,20 @@ trait EvenBetterPathMethods {
     newInputStream
   }
 
-  def writeContent(content: String)(openOptions: OpenOptions, codec: Codec)(implicit ec: ExecutionContext): this.type = {
+  def gzipByteArray(byteArray: Array[Byte]): Array[Byte] = {
```

Comment:
should this be `private`?

Reply:
Yes, thanks. Actually it should be `protected`, since it's used in the subclass